lumicks/pylake/kymotracker/tests/test_greedy_algorithm.py

```diff
@@ -61,6 +62,13 @@ def test_kymotracker_greedy_algorithm_integration_tests(kymo_integration_test_da
     np.testing.assert_allclose(tracks[0].time_idx, np.arange(15, 25))
 
 
+def test_greedy_algorithm_empty_result(kymo_integration_test_data):
+    test_data = kymo_integration_test_data
+    tracks = track_greedy(test_data, "red", track_width=300, pixel_threshold=400)
```

Comment:
It is not immediately clear that you will get an empty result due to absurd parameters: they aren't absurd enough.
Perhaps add a code comment?

Reply:
I've added a small comment :)

Reply:
I'm fine with it, doesn't hurt to leave it in. Was not sure if you included it purposefully.